Sources/MongoSwift/MongoCursor.swift

```diff
@@ -4,7 +4,7 @@ import mongoc
 public class MongoCursor<T: Codable>: Sequence, IteratorProtocol {
     internal var _cursor: OpaquePointer?
     private var _client: MongoClient?
-    internal var _session: ClientSession?
+    private var _session: ClientSession?
```

Comment:
are `_client` and `_session` `fileprivate` now?

Reply:
they're `private` - we only use both of them within `MongoCursor` which is all contained in this file. 
`fileprivate` would be mildly more permissive and allow them to be used within this file outside of the class definition